app/Http/Controllers/TodoController.php

```diff
@@ -5,117 +5,79 @@
 use Illuminate\Http\Request;
 use App\Models\Todo;
 use Illuminate\Support\Facades\Auth;
-use App\Models\User;
+use App\Http\Requests\TodoRequest;
 
 class TodoController extends Controller
 {
     public function index()
     {
-        if(!Auth::check()){
-            $msg = "Login First";
-            return $this->responseNotfound($msg);
-        }
+        $todos = Auth::user()->todos;//return type: collection
 
-        $todos = User::find(Auth::id())->todos;//return type: collection
         //If  I don't add values()->all(), it will return itself generated indices, so I can't return the correct format
         //use the values method to reset the keys to consecutively numbered indexes:
         $sorted = $todos->sortByDesc('created_at')->values()->all();//To show recently created first
-        
+
         return $this->responseSuccess($sorted);
     }
 
-    public function store(Request $request)
+    public function store(TodoRequest $request)
     {
-        if(!Auth::check()){
-            $msg = "Login First";
-            return $this->responseNotfound($msg);
-        }
         //사용자의 입력 데이터 유효성 검사
-        $request->validate([
-            'name' => 'required | max:255',
-            'description' => 'required | max:1000',
-        ]);
-
-        $data = $request->all();//取得所有input
-
-        //It doesn't throw an error to me.
-        try{
-            $todo = Todo::create([
-                'user_id' => Auth::id(),
-                'name' => $data['name'],
-                'description' => $data['description']
-            ]);
-        } catch(\Exception $e){
-            $this->responseFail($e->getMessage());
-        }
+        $data = $request->post();
+    
+        //Handled by Exceptions/Handler
+        $todo = Todo::create([
+            'user_id' => Auth::id(),
+            'name' => $data['name'],
+            'description' => $data['description']
+        ]);  
 
         return $this->responseSuccess($todo);
     }
 
     public function details(int $id)
     {
-        if(!Auth::check()){
-            $msg = "Login First";
-            return $this->responseNotfound($msg);
-        }
-        $todo = User::find(Auth::id())->todos->find($id);
+        $todo = Auth::user()->todos->find($id);
 
-        //Fail to find a data
-        if(!$todo){
+        //Fail to find the data
+        if (!$todo){
```

Comment:
大括號前要空格，其他相似地方可以一併調整
```suggestion
        if (!$todo) {
```